src/db/util/raw-model.ts

```diff
@@ -73,6 +75,14 @@ export type DestroyFn<F extends FieldDefinition> = (args: {
 
 export type TruncateFn = (trx?: Knex.Transaction<any, any>) => Promise<void>;
 
+export type CountFn<F extends FieldDefinition, AdditionalArgs = {}> = (
+  args?: {
+    where?: WhereCond<F>;
+    trx?: Knex.Transaction<any, any>;
+    distinct?: Array<keyof InstanceDataOf<F>>;
+  } & AdditionalArgs
+) => Promise<number>;
```

Comment:
Since PostgreSQL returns `bigint` type, type which will be represented as a string and not a number, as can be seen in both [PostgreSQL](https://www.postgresql.org/docs/16/functions-aggregate.html) and [Knex docs](https://knexjs.org/guide/query-builder.html#count), I would convert back the string to [`BigInt` type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt), since it now natively exists in Javascript.
```suggestion
) => Promise<bigint>;
```